src/en/hentairead/src/eu/kanade/tachiyomi/extension/en/hentairead/Hentairead.kt

```diff
@@ -76,4 +76,15 @@ class Hentairead : Madara("HentaiRead", "https://hentairead.com", "en", dateForm
             ),
         )
     }
+
+    override fun pageListRequest(chapter: SChapter): Request {
+        // There's like 2 non-English entries where this breaks
+        chapter.url = "${chapter.url}english/p/1/"
+        return super.pageListRequest(chapter)
```

Comment:
dont update url in schapter, I belive j2k ends up updating the url in db when this happens. (at least for smanga url)

Reply:
I checked J2K and it seems to work fine. It is in any case desirable to propagating changes if they decide to change the behavior in the future. I'll copy the object over instead.